### PR TITLE
feat: add monaco docs CSS override

### PR DESCRIPTION
**Pull request by wusteven815**
- Adds #2247 
- Adds a few CSS overrides to Monaco

### CODE REVIEW

**Comment by wusteven815** [comment]
- `min-width` - it should have a min width to avoid looking thin
- `hr` - make horizontal breaks visible
- `blockquote` - make block quotes have a margin to make them indented

File: packages/console/src/ConsoleInput.scss [hunk 1]
```diff
@@ -74,3 +74,16 @@ $console-input-margin-color: $foreground;
     }
   }
 }
+
+.docs {
```
Comment (dsmmcken) [inline packages/console/src/ConsoleInput.scss:78]: That's a pretty greedy selector, and not scoped. Where is ".docs" as the class defined?

I think Editor.css is probably a better place for this as well.

File: packages/console/src/ConsoleInput.scss [hunk 2]
```diff
@@ -74,3 +74,16 @@ $console-input-margin-color: $foreground;
...
+  min-width: 200px;
```
Comment (dsmmcken) [inline packages/console/src/ConsoleInput.scss:79]:
```suggestion
  min-width: 12rem;
```

File: packages/console/src/ConsoleInput.scss [hunk 3]
```diff
@@ -74,3 +74,16 @@ $console-input-margin-color: $foreground;
...
+  
+  hr {
+    background-color: $text-muted;
+    height: 1px;
+  }
+
+  blockquote {
+    margin-left: 20px;
```
Comment (dsmmcken) [inline packages/console/src/ConsoleInput.scss:87]:
```suggestion
    margin-left: $spacer-3;
```

Or spacer-2 whichever is closer.

**Review by dsmmcken — DISMISSED** [review]

File: packages/console/src/notebook/Editor.scss [hunk 4]
```diff
@@ -15,3 +15,16 @@
     color: $link-color;
   }
 }
+
+.rendered-markdown.markdown-docs {
```
[the remark this answers is not in the document]
Comment (wusteven815) [reply packages/console/src/notebook/Editor.scss:19]: Yes, this is the selector for single element with `.rendered-markdown` and `.markdown-docs` classes

**Review by dsmmcken — APPROVED** [review]

**Review by dsmmcken — CHANGES_REQUESTED** [review]
Actually, one last thing. Comment what this is for. In future I could see a case of someone searching this codebase and not finding any matches for it so it could accidentally get deleted without knowing it originates elsewhere.

File: packages/console/src/notebook/Editor.scss [hunk 5]
```diff
@@ -16,6 +16,7 @@
   }
 }
 
+// Override Monaco's default markdown popups (i.e signature help) to resemble VS Code closer
```
Comment (dsmmcken) [inline packages/console/src/notebook/Editor.scss:19]: Please specifically mention that this class is defined in core or wherever.

**Review by dsmmcken — APPROVED** [review]

**Pull request merged**